<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Kunden</class>
 <widget class="QDialog" name="frm_Kunden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kunden</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>557</width>
     <height>494</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Kontakt">
    <attribute name="title">
     <string>Kontakt</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="lb_Firma">
         <property name="text">
          <string>Firma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_Firma"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lb_Strasse">
         <property name="text">
          <string>Strasse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_Strasse"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lb_Adresse">
         <property name="text">
          <string>Land/PLZ/Ort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="land"/>
       </item>
       <item>
        <widget class="QLineEdit" name="le_PLZ"/>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ort"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Telefon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_telefon"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>E-Mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_mail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Internet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_Internet"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Speichern">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_Sonstiges">
    <attribute name="title">
     <string>Sonstiges</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_Internes">
    <attribute name="title">
     <string>Internes</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_Artikel">
    <attribute name="title">
     <string>Artiekel</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>290</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kunden Nr.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
